test/e2e/e2eutils.go

```diff
@@ -0,0 +1,40 @@
+package e2e
+
+import (
+	goctx "context"
```

Comment:
Why alias? There is no other context here, right?

Reply:
@baijum alias can be removed